contracts/timed_asset_lock_contract/program.py

```diff
@@ -68,7 +78,6 @@ def closeAccountTo(account: Expr) -> Expr:
         Assert(
             And(
                 # The unlock timestamp must be at some point in the future.
-                Global.latest_timestamp() < on_create_unlock_time,

```

Comment:
Was this intentional?

Reply:
Yes, as at this point in the contract you haven't committed anything to the escrow, no need to check timestamp here....I took it out some place else which allowed an edge condition where your funds would be permanantly locked up if you had a small time period and you didn't complete all the required app calls in time